nautobot/dcim/templates/dcim/softwareimagefile_retrieve.html

```diff
@@ -87,11 +87,25 @@
                                 <tr>
                                     <td>Device Types</td>
                                     <td>
-                                        {% if object.device_types.count %}
-                                        <a href="{% url 'dcim:devicetype_list' %}?software_images={{ object.pk }}">{{ object.device_types.count }}</a>
-                                        {% else %}
-                                        <span class="text-muted">&mdash;</span>
-                                        {% endif %}
+                                        {% with device_type_count=object.device_types.count %}
+                                            {% if device_type_count %}
+                                            <a href="{% url 'dcim:devicetype_list' %}?software_image_files={{ object.pk }}">{{ device_type_count }}</a>
+                                            {% else %}
+                                            <span class="text-muted">&mdash;</span>
+                                            {% endif %}
+                                        {% endwith %}
+                                    </td>
+                                </tr>
+                                <tr>
+                                    <td>Devices</td>
```

Comment:
Wonder if something like `Devices overridden to use this file` might be clearer?